api-spec/extensions/version/README.md

```diff
@@ -0,0 +1,150 @@
+# Version API Extension
+
+**Extension [Maturity Classification](../../../extensions/README.md#extension-maturity): Pilot**
+
+The core API doesn't support semantics to creating and accessing different versions of an item or collection record. This version API extension defines the API resources and semantics for creating and accessing versioned records.
+
+## Methods
+
+| Path                                                  | Content-Type Header | Description |
+| ----------------------------------------------------- | ------------------- | ----------- |
+| `GET /collections/{collectionID}/versions`              | `application/json`  | Returns a catalog response with links to all versions of a given collection. |
+| `GET /collections/{collectionID}/versions/{versionId}`              | `application/json`  | Returns a collection record. |
+| `GET /collections/{collectionID}/items/{featureId}/versions/`              | `application/json`  | Returns a catalog response with links to all versions of a given item. |
+| `GET /collections/{collectionID}/items/{featureId}/versions/{versionId}`              | `application/json`  | Returns an item record. |
+
+## How It Works
+
+When this extension is implemented, the API supports `versions` resource that list all versions of an item or collection and provides permanent links to each version.
+
+The latest version of an item is accessible at `/collections/{collectionID}/items/{itemsId}`. The record has a link with `"rel": "permalink"` that links to the permanent location of the record which is `/collections/{collectionID}/items/{itemsId}/versions/{versionId}`.
+
+If the record has a previous version it also provides a link to the permanent location of that version using `"rel": "previous_version"`
```

Comment:
The registered rel type `predecessor-version` would be better, I think. Would also better fit with the relation type successor-version, which is used with the deprecated flag.